.metadocrc

```diff
@@ -0,0 +1,15 @@
+{
+	"files": [
+	    "impress.js",
+	    "lib/client.js"
+	],
+	"customLinks": {
+	    "Application": "https://github.com/metarhia/impress/blob/master/lib/application.js",
+	    "IncomingMessage": "https://nodejs.org/dist/latest-v10.x/docs/api/http.html#http_class_http_incomingmessage",
+        "ServerResponse": "https://nodejs.org/dist/latest-v10.x/docs/api/http.html#http_class_http_serverresponse",
```

Comment:
```suggestion
       "ServerResponse": "https://nodejs.org/dist/latest-v10.x/docs/api/http.html#http_class_http_serverresponse",
```
?

Reply:
Weird stuff, in editor everything looks fine, so I've missed this here 
![image](https://user-images.githubusercontent.com/31672093/52533080-3a88b600-2d37-11e9-87cd-97a1f1626fde.png)